plugins/convertPathData.js

```diff
@@ -638,6 +643,28 @@ function filters(
         }
       }
 
+      // round arc radius more accurately
+      // eg m 0 0 a 1234.567 1234.567 0 0 1 10 0 -> m 0 0 a 1235 1235 0 0 1 10 0
+      if (
+        params.smartArcRounding &&
+        command === 'a' &&
+        sagitta !== undefined &&
+        precision
+      ) {
+        for (let precisionNew = precision; precisionNew >= 0; precisionNew--) {
+          const radius = toFixed(data[0], precisionNew);
+          const sagittaNew = /** @type {number} */ (
+            calculateSagitta([radius, radius].concat(data.slice(2)))
```

Comment:
```suggestion
            calculateSagitta([radius, radius, ...data.slice(2)])
```

Subjective, just think it looks nicer.